### ISSUE TITLE
Is it possible to record audio and motion acceleration at the same time?

### ISSUE THREAD

**Issue by scheung38** [syedhali/EZAudio#160]
Thanks in advance. 

Also could you convert your project into Swift?

**Comment by syedhali** [comment]
Not too sure about this one, is the motion acceleration using the microphones for anything?

**Comment by scheung38** [comment]
No, just record audio using EZAudio with mic and at the same time record motion using Accelerometer and then get snapshot of dB and motion data once recording done
